smaily_for_drupal/src/Plugin/Block/SmailyForDrupalFormBlock.php

```diff
@@ -68,12 +95,33 @@ public function build() {
     $config = [
       'autoresponder' => $this->configuration['smaily_autoresponder'],
       'button_title' => $this->configuration['smaily_button_title'],
+      'success_url' => $this->configuration['smaily_success_url'] ?: $this->getSuccessPath(),
+      'failure_url' => $this->configuration['smaily_failure_url'] ?: $this->getFailurePath(),
     ];
     $form = \Drupal::formBuilder()->getForm('Drupal\smaily_for_drupal\Form\SubscribeForm', $config);
-
     return $form;
   }
 
+  /**
+   * Get full URL to default success response page.
+   *
+   * @return string
+   *   URL i.e. https://localhost/drupal/smaily_for_drupal/success
+   */
+  public function getSuccessPath() {
+    return Url::fromUri('base:/smaily_for_drupal/success', ['absolute' => TRUE, 'https' => TRUE])->toString();
```

Comment:
I think you shouldn't enforce URL to use HTTPS, because a site might not be using HTTPS. Rather omit the `'https' => TRUE` option and let the `Url` class handle it for you.

From the [documentation](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Url.php/function/Url%3A%3AfromUri/8.2.x):
> If not defined, the current scheme is used, so the user stays on HTTP or HTTPS respectively